maro/simulator/scenarios/finance/README.md

```diff
@@ -0,0 +1,149 @@
+# Finance scenario in MARO
+
+## Introduction
+
+The Finance scenario simulates securities trading in financial markets.
+Imagine you invest in a finance market. For example, CN Shanghai:
+You can create a strategy which decides how to buy and sell stocks.
+So, your investment can keep increasing no matter how the market goes up or down.
+
+## Environment Modeling
+
+To simulate real markets, we implement the typical order types, cost of the trading.
```

Comment:
The typical order types are supported, such as .... To mimic the real market, we also have the internal logic to handle slippage, tax ....